.github/workflows/check.yaml

```diff
@@ -32,6 +32,10 @@ jobs:
       matrix:
         juju-channel: ['3.4/stable', '3.5/stable']
         command: ['TEST_JUJU3=1 make functional']
+        include:
+          - juju-channel: '2.9/stable'
+            command: 'make functional'
```

Comment:
Please consider how this will work with the automation repo template. You may need to restructure the template a bit to support this.

Reply:
That's true, but the point here is to test 2.9/stable and add back bionic. Porting back to automation repo can be done separately (or do we still need this support this charm?). This charm maybe replaced by node-exporter

Reply:
Sure, no worries. :+1: Porting to automation should at least be done in the same ticket though, otherwise the automation will try reverting these changes when it runs next.

Reply:
No worries, we can figure out automation later.

Reply:
Follow up, inactive != not maintained, will use https://github.com/canonical/hardware-observer-operator/pull/342#discussion_r1846232863 as a solution (try to make things consistent)